pkg/kubelet/images/image_manager.go

```diff
@@ -18,6 +18,7 @@ package images
 
 import (
 	"fmt"
+	"k8s.io/kubernetes/pkg/kubelet/metrics"
```

Comment:
By the way, you should group the import list.  `std` --> `github/xxx` --> `k8s.io/xxx`

Reply:
thanks for reminding, fixed